scene/gui/graph_edit.cpp

```diff
@@ -260,6 +260,8 @@ PackedStringArray GraphEdit::get_configuration_warnings() const {
 }
 
 Error GraphEdit::connect_node(const StringName &p_from, int p_from_port, const StringName &p_to, int p_to_port) {
+	ERR_FAIL_COND_V_MSG(connections_layer == nullptr, ERR_BUG, "connections_layer is missing.");
```

Comment:
Note that `ERR_BUG` should only be used for conditions which should _never_ happen, as it's meant to communicate an engine bug. If this can be caused by a user misusing the API, it should have a different error code.

Reply:
`FAILED` or `ERR_UNAVAILABLE` might be better

Reply:
`ERR_UNAVAILABLE` is for unimplemented APIs.

Reply:
Changed to `FAILED`. 
But I don't know why this method returns an `Error` at all. Feels a bit out of place here and always returned `OK`.